### DISCUSSION TITLE
How to visualize the structure of a trained model and obtain its corresponding parameters ?

### DISCUSSION THREAD

**Discussion by Dexterssssss** [autogluon/autogluon#4509] [Q&A]
AutoGluon is such a great tool, especially its automated multi-model training capability. I've been trying to visualize the structure of trained models and obtain the parameters of each layer over the past few days. After reviewing the official documentation and browsing numerous websites, I still haven't found a solution. Does anyone have a good method?

**Comment by ghost** [comment]
I'm also looking for this
